stable/console-chart/templates/console-clusterrole.yaml

```diff
@@ -79,7 +84,11 @@ rules:
   - hive.openshift.io
   resources:
   - clusterdeployments
+  - clusterpools
   - clusterimagesets
+  - clusterprovisions
+  - clusterdeprovisions
```

Comment:
Currently we grab the ClusterProvisions for the provision failure reason; I think there might be some information in the ClusterDeprovision we might be able to leverage